resources/migrations/001_update_migrations.yaml

```diff
@@ -9281,7 +9281,7 @@ databaseChangeLog:
       changes:
         - createTable:
             tableName: notification
-            remarks: join table that connect notification subscriptions and notification destinations
+            remarks: join table that connect notification subscriptions and notification handlers
```

Comment:
I guess its fine to leave this rename here, and keeping as "destinations" in the previous PR.

Reply:
Hmmm, actually I wonder if this is what's breaking the checksum in the rollback tests. Silly for the comment to be in the hash 😑

Reply:
Yeah, it's what is breaking the checksum!